working_notebook-TO_COMPLETE_Anna.ipynb

```diff
@@ -815,19 +815,25 @@
   },
```

Comment:
<p>Try to give it a go to the testing exercise too! </p><p> </p><p>That said: <strong>Congratulations</strong> for the amazing job you've done so far! 🎉 </p><p>and <strong>Thanks</strong> very much for sending your answers! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WebValley2021ReImagined/programming-for-data-science/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>